### PR TITLE
Revert "Update releases.json"

### PR DESCRIPTION
**Pull request by chaos-ui**
Reverts dotnet/core#6166

### CODE REVIEW

**Review by chaos-ui — COMMENTED** [review]
Shouldn't this be merged?

**Comment by rbhanda** [comment]
Why are we reverting this change?

**Comment by chaos-ui** [comment]
Im not spam you guys

File: release-notes/2.1/releases.json [hunk 1]
```diff
@@ -242,7 +242,7 @@
           "runtime-version": "2.1.27",
           "vs-version": "15.9.35",
           "vs-mac-version": "",
-          "vs-support": "Visual Studio 2017 (v15.9)",
+          "vs-support": "Visual Studio 2019 (v15.9)",
           "vs-mac-support": "",
```
Comment (chaos-ui) [inline release-notes/2.1/releases.json:246]: update to docs "2019 Visual studio from 2017

**Pull request closed without merging**